modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -29,21 +29,31 @@ export default ({fields}) => {
 		}
 	});
 
+	const setSearchParamsWithoutPageReload = (param, index) => {
+		const url = new URL(location.toString());
```

Comment:
@brunofariasdeo, please add a blank line after the variables declarations and the use of them. 😉

In addition, can you try to solve this issue without use the `URL` object? Unfortunately, it's not supported by IE, as you can see on https://developer.mozilla.org/en-US/docs/Web/API/URL/URL.